ui/core-react/src/core-react/searchbox/SearchBox.tsx

```diff
@@ -84,26 +83,20 @@ export class SearchBox extends React.Component<SearchBoxProps, SearchBoxState> {
           onKeyDown={this._handleKeyDown}
           onPaste={this._trackChange}
           onCut={this._trackChange}
-          placeholder={
-            this.props.placeholder
-              ? this.props.placeholder
-              : UiCore.translate("general.search")
-          }
```

Comment:
Shouldn't placeholder be passed here as `placeholder={this.props.placeholder}`? You added the property to `SearchBoxInput` but don't pass it.

Reply:
Yes, definitely, good catch!